src/components/Tile.scss

```diff
@@ -7,15 +7,18 @@ $tile-size: 80px;
 }
 
 .tile {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
   position: absolute;
-  width: $tile-size;
-  height: $tile-size;
+  width: calc(((100% - 20px) / 4 - 20px));
```

Comment:
`box-sizing: border-box` in many cases allows to handle margins without `calc`: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing